Cliqz/Extensions/AppDelegateExtension.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  AppDelegateExtension.swift
+//  Client
+//
+//  Created by Tim Palade on 5/25/18.
+//  Copyright © 2018 Cliqz. All rights reserved.
+//
+
+import UIKit
+
+let InstallDateKey = "InstallDateKey"
+#if GHOSTERY
+let HasRunBeforeKey = "previous_version"
+#endif
+extension AppDelegate {
+    func recordInstallDateIfNecessary() {
+        guard let profile = self.profile else { return }
+        if profile.prefs.stringForKey(LatestAppVersionProfileKey)?.components(separatedBy: ".").first == nil {
```

Comment:
why didn't you use `SettingsPrefs` instead of profile.prefs

Reply:
Because this is not a Setting Pref. Those are reserved for the settings. This is not a setting.

Reply:
yeah, you are right